tests/test_models/test_losses/test_feature_loss.py

```diff
@@ -0,0 +1,52 @@
+import pytest
+import torch
+
+from mmedit.models.losses import FeatureLoss
+
+
+def test_feature_loss():
+    pretrained = 'https://download.openmmlab.com/mmediting/' + \
```

Comment:
There may be a problem loading checkpoints during test. Need confirm with @innerlee